OSDOCS-5274: Restart node when changing mtu value (#55763)
openshift/openshift-docs: modules/microshift-restart-ovnkube-master.adoc

> // Module included in the following assemblies:

Reviewer: Is this restarting of ovnkube-master pod only referenced by mtu change or is it also referenced by the firewall known issue?
If it is for mtu, then we probably don't need it since node reboot will restart ovnkube-master.

Author: @zshi-redhat OK, great, thank you. That was the question I had asked above (do we need to do both).

//
// * microshift_networking/microshift-networking.adoc

:_content-type: PROCEDURE
[id="microshift-restart-ovnkube-master_{context}"]
= Restarting the ovnkube-master pod

> The following procedure restarts the ovn-kube master pod.

Reviewer: In the case this file is needed, let's use the name `ovnkube-master pod` everywhere.

> The following procedure restarts the ovn-kube master pod, which also restarts the node.

Reviewer 2: "which also restarts the node" is not accurate. The steps below only restart the ovn-master pod. A node refers to a virtual or physical machine. Whereas a pod is a set of running containers. Containers run on a node.

Author: Ah OK, I thought the ovnkube-master restart was making the node restart so that it found the changed value. (Following the earlier comments on the Jira by @zshi-redhat about restarting the node.) So we need to do two things: 1) Restart ovnkube to adopt the value change, 2) then reboot the host to apply the new packet size to all the pods?

.Prerequisites

* The OpenShift CLI (`oc`) is installed.
* Access to the cluster as a user with the `cluster-admin` role.
* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
* The KUBECONFIG environment variable is set.

.Procedure

> Run the commands listed in each step that follows to restart ovnkube-master pod.

Reviewer 3: Should this be 
```suggestion
Run the commands listed in each step that follows to restart `ovnkube-master` pod.
```
And all other references to it

Reviewer 3: Also, I have a suggested rewrite for clarity if you would like. I got a little confused because each step says "run the following command"  you could honestly remove "listed in each step" and i think that would help with the word redundancy. So,
```suggestion
Run the following commands to restart the ovnkube-master pod.
```

or 

```suggestion
Follow the following steps to restart the ovnkube-master pod.
```

Author: I went with "Use the following steps."

> Run the commands listed in each step that follows to restart oven-kube master pod.

Reviewer: ```suggestion
Run the commands listed in each step that follows to restart ovnkube-master pod.
```

> Run the commands listed in each step that follows to restart the node.

Reviewer 2: ...to restart the "ovnkube-master pod".  Not "Node"

. Access the remote cluster by running the following command:
+
[source, terminal]
----
$ export KUBECONFIG=$PWD/kubeconfig
----

. Find the name of the ovnkube-master pod that you want to restart by running the following command:
+
[source, terminal]
----
$ pod=$(oc get pods -n openshift-ovn-kubernetes | awk -F " " '/ovnkube-master/{print $1}')
----

. Delete the ovnkube-master pod by running the following command:
+
[source, terminal]
----
$ oc -n openshift-ovn-kubernetes delete pod $pod
----

> . Confirm that a new ovnkube-master pod has been started by running the following command:

Reviewer 3: ```suggestion
. Confirm a new ovnkube-master pod has started by running the following command:
```

+
[source, terminal]
----
$ oc get pods -n openshift-ovn-kubernetes
----

> The listing of the running pods shows a new ovnkube-master pod name and age.

Reviewer 3: This sentence seems out of place. Could this be a verification step maybe?

Author: What we really need here is an example of the output when new pods launch. I haven't had time to add that.

// Module included in the following assemblies:
//
// * microshift_networking/microshift-networking.adoc

:_content-type: PROCEDURE
[id="microshift-restart-ovnkube-master_{context}"]

> == Restarting the ovnkube-master pod

Reviewer 4: ```suggestion
= Restarting the ovnkube-master pod
```

The following procedure restarts the ovn-kube master pod, which also restarts the node.

.Prerequisites

* The OpenShift CLI (`oc`) is installed.
* Access to the cluster as a user with the `cluster-admin` role.
* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
* The KUBECONFIG environment variable is set.

> .Procedure

Reviewer 4: You shouldn't have two procedure sections.
